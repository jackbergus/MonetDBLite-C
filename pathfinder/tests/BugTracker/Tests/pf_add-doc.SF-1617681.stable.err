stderr of test 'pf_add-doc.SF-1617681` in directory 'tests/BugTracker` itself:


# 18:41:57 >  
# 18:41:57 >  Mtimeout -timeout 180 Mserver "--config=/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/ufs/manegold/_/scratch0/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./PATHFINDER/lib64/MonetDB:/ufs/manegold/_/scratch0/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./PATHFINDER/lib64/MonetDB/lib:/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/lib64/MonetDB:/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/lib64/MonetDB/lib:/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/lib64/MonetDB/bin" --set "gdk_dbfarm=/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/var/MonetDB/dbfarm" --set "sql_logdir=/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/var/MonetDB/sql_logs" --set "xquery_logdir=/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/var/MonetDB/xquery_logs" --set mapi_port=36142 --set monet_prompt= --trace "--dbname=mTests_tests_BugTracker" --dbinit=" module(pathfinder);" ; echo ; echo Over..
# 18:41:57 >  


# 18:41:57 >  
# 18:41:57 >  Mtimeout -timeout 60 MapiClient -lxquery -fxml -umonetdb -Pmonetdb --host=localhost --port=36142 pf_add-doc.SF-1617681_s00.xq
# 18:41:57 >  


# 18:41:57 >  
# 18:41:57 >  Mtimeout -timeout 60 MapiClient -lxquery -fxml -umonetdb -Pmonetdb --host=localhost --port=36142 pf_add-doc.SF-1617681_s01.xq
# 18:41:57 >  


# 18:41:57 >  
# 18:41:57 >  Done.
# 18:41:57 >  

