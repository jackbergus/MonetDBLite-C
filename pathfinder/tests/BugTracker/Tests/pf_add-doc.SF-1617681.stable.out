stdout of test 'pf_add-doc.SF-1617681` in directory 'tests/BugTracker` itself:


# 20:08:39 >  
# 20:08:39 >  Mtimeout -timeout 180 Mserver "--config=/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/ufs/manegold/_/scratch0/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./PATHFINDER/lib64/MonetDB:/ufs/manegold/_/scratch0/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./PATHFINDER/lib64/MonetDB/lib:/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/lib64/MonetDB:/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/lib64/MonetDB/lib:/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/lib64/MonetDB/bin" --set "gdk_dbfarm=/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/var/MonetDB/dbfarm" --set "sql_logdir=/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/var/MonetDB/sql_logs" --set "xquery_logdir=/net/corona.ins.cwi.nl/export/scratch0/manegold/Monet/Testing/Stable/.GNU.64.64.d.--disable-debug_--enable-assert_--enable-optimize.PREFIX./MONETDB/var/MonetDB/xquery_logs" --set mapi_port=39716 --set monet_prompt= --trace "--dbname=mTests_tests_BugTracker" --dbinit=" module(pathfinder);" pf_add-doc.SF-1617681.prologue ; echo ; echo Over..
# 20:08:39 >  

# Monet Database Server V4.14.0
# Copyright (c) 1993-2006, CWI. All rights reserved.
# Compiled for x86_64-redhat-linux-gnu/64bit with 64bit OIDs; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.

printf("\nReady.\n");

Ready.
quit();

Over..

# 20:08:39 >  
# 20:08:39 >  Mtimeout -timeout 60 MapiClient -lxquery -oxml -umonetdb -Pmonetdb --host=localhost --port=39716 pf_add-doc.SF-1617681_s00.xq
# 20:08:39 >  


# 20:08:39 >  
# 20:08:39 >  Mtimeout -timeout 60 MapiClient -lxquery -oxml -umonetdb -Pmonetdb --host=localhost --port=39716 pf_add-doc.SF-1617681_s01.xq
# 20:08:39 >  

<?xml version="1.0" encoding="utf-8"?>
<XQueryResult><x/><x/><x/><x/></XQueryResult>

# 20:08:39 >  
# 20:08:39 >  Done.
# 20:08:39 >  

